diff -urN smartmontools-5.42/smartd.service.in smartmontools-5.42-patched/smartd.service.in
--- smartmontools-5.42/smartd.service.in	2012-01-10 12:52:35.627980195 +0400
+++ smartmontools-5.42-patched/smartd.service.in	2012-01-10 12:53:13.842502505 +0400
@@ -3,8 +3,8 @@
 After=syslog.target
 
 [Service]
-EnvironmentFile=/usr/local/etc/sysconfig/smartmontools
-ExecStart=/usr/local/sbin/smartd -n $smartd_opts
+EnvironmentFile=/etc/sysconfig/smartd
+ExecStart=/usr/sbin/smartd -n $smartd_opts
 
 [Install]
 WantedBy=multi-user.target
