diff -up smartmontools-7.0/smartd.service.in.1~ smartmontools-7.0/smartd.service.in
--- smartmontools-7.0/smartd.service.in.1~	2019-01-14 02:35:03.656845086 +0100
+++ smartmontools-7.0/smartd.service.in	2019-01-14 02:36:53.237477004 +0100
@@ -4,8 +4,8 @@ Documentation=man:smartd(8) man:smartd.c
 
 [Service]
 Type=notify
-EnvironmentFile=-/usr/local/etc/sysconfig/smartmontools
-ExecStart=/usr/local/sbin/smartd -n $smartd_opts
+EnvironmentFile=-/etc/sysconfig/smartd
+ExecStart=/usr/sbin/smartd -n $SMARTD_OPTIONS
 ExecReload=/bin/kill -HUP $MAINPID
 StandardOutput=syslog
 
